src/store/ChartStore.js

```diff
@@ -445,6 +445,8 @@ class ChartStore {
 
     // Calling newChart with symbolObj as undefined refreshes the chart
     @action.bound newChart(symbolObj = this.currentActiveSymbol, params) {
+        if(!symbolObj) 
```

Comment:
this changes the logic altogether. Setting `symbolObj` to undefined should cause the chart to refresh, not do nothing.

actually I think there's a bug with `refreshChart`. The implementation should be `this.newChart(undefined)`.

Reply:
@brucebinary im not sure, but as far as i understood if `symbolObj` is `undefind` we are using `this.currentActiveSymbol` we do nothing which means the `chart`  is not completly initialized so we dont need realy to refresh, i needed  `refreshchart` only to because when connection closes we forget all the streams so this only is realy needed there which `currentActiveSymbol` has value so it refresh the chart.